docs/proposals/20210310-opt-in-autoscaling-from-zero.md

```diff
@@ -229,6 +265,8 @@ metadata:
       capacity.cluster-autoscaler.kubernetes.io/memory: "500mb"
       capacity.cluster-autoscaler.kubernetes.io/cpu: "1"
       capacity.cluster-autoscaler.kubernetes.io/ephemeral-disk: "100Gi"
+      node-info.cluster-autoscaler.kubernetes.io/cpu-architecture: "arm64"
+      node-info.cluster-autoscaler.kubernetes.io/os: "linux"
```

Comment:
Hm the suffix here matches neither the kubernetes.io/ labels nor the fields in the NodeInfo struct. I think it would make sense to align to the NodeInfo struct field names

Reply:
Done.